assets/js/admin/orders/order-overview/views/form-add-order-adjustment.js

```diff
@@ -201,10 +201,13 @@ export const FormAddOrderAdjustment = Dialog.extend( {
 			orderItem.get( 'adjustments' ).add( model );
 			// Adding to the Collection doesn't bubble up a change event.
 			orderItem.trigger( 'change' );
+			model.set( 'objectType', 'order_item' );
```

Comment:
@spencerfinnell if you have a chance, could you put eyes on this? I changed it as the template markup was always setting the adjustment to be `order`, even for `order_item` specific fees, but I don't trust my JS here and may have broken things.

Reply:
@robincornett The code itself looks correct. If it's performing as expected, I think you're good to go.